anyway/infographics_utils.py

```diff
@@ -35,18 +96,40 @@
                  'rank': int (Integer)
                  }
     }
-"""
+    """
+    def __init__(self, request_params: RequestParams):
+        self.request_params: RequestParams = request_params
```

Comment:
This is nonstandard. Put the declaration on th class level, without initialization:
```python
class Widget:
    request_params: RequestParams
    name: str
```
etc.

Reply:
So the fields will be listed twice? (declaration and then initialization)

Reply:
Yes, that is the standard way - much like most other OO languages :)